src/planner/binder/expression/bind_lambda.cpp

```diff
@@ -156,4 +156,19 @@ void ExpressionBinder::CaptureLambdaColumns(vector<unique_ptr<Expression>> &capt
 	}
 }
 
+idx_t ExpressionBinder::MatchingLambdaBinding(ColumnRefExpression &expr) const {
+	if (lambda_bindings == nullptr) {
+		return DConstants::INVALID_INDEX;
+	}
+	auto &table_name = expr.GetTableName();
+	auto &bindings = (vector<DummyBinding> &)*lambda_bindings;
+	for (idx_t i = 0; i < bindings.size(); i++) {
```

Comment:
Nit: maybe a range-based for loop?

Reply:
I have to return `i` so I don't think I can use a range based for loop here